### DISCUSSION TITLE
Options "Randomly collapsing...mines take some money instead" of what?

### DISCUSSION THREAD

**Discussion by capsicumw** [Keriew/augustus#990] [Q&A]
In the configuration options, city management tab, I see "Randomly collapsing clay pits and iron mines take some money instead". Instead of what? Is this a minor bug [typo, however you like to classify it]? The manual also has an incomplete sentence and no further explanation.

**Comment by xyn22** [comment]
> In the configuration options, city management tab, I see "Randomly collapsing clay pits and iron mines take some money instead". Instead of what? Is this a minor bug [typo, however you like to classify it]? The manual also has an incomplete sentence and no further explanation.

Instead of "collapsing" in which case you'd have to clear the rubble and build again, sometimes it's needlessly distracting hence the significant option.

I think because "collapsing" is a part of the statement the word "instead" explicitly implies the negation of "collapsing". I don't think it needs more context to be understood. If it was "instead of collapsing" it would seem redundant to repeat the word. But I'm interested, what would you suggest to be a better explanation?

**Comment by capsicumw** [comment]
The sentence structure of "Randomly collapsing clay pits and iron mines take some money instead", implies that they still collapse and taking money is related to the collapse. 
I suggest "Clay pits and iron mines randomly take some money instead of collapsing to rubble." 
Or "Instead of randomly collapsing, clay pits and iron mines take some money"

**Comment by Ouaz** [comment]
Ah, there's actually a PR about that, not about the wording, but the length.
https://github.com/Keriew/augustus/pull/992

Max glyphs per line to prevent cut line is 63.

What about: `Pits and mines randomly take some money instead of collapsing`

**Reply by capsicumw** [reply]
That looks like a fine sentence.
